airbyte-integrations/connectors/source-twilio-taskrouter/source_twilio_taskrouter/schemas/workers.json

```diff
@@ -1,47 +1,47 @@
 {
-  "type": "object",
-  "properties": {
-    "account_sid": {
-      "type": ["null", "string"]
-    },
-    "date_created": {
-      "type": ["null", "string"],
-      "format": "date-time"
-    },
-    "date_updated": {
-      "type": ["null", "string"],
-      "format": "date-time"
-    },
-    "workspace_sid": {
-      "type": ["null", "string"]
-    },
-    "attributes": {
-      "type": ["null", "string"]
-    },
-    "date_status_changed": {
-      "type": ["null", "string"],
-      "format": "date-time"
-    },
-    "friendly_name": {
-      "type": ["null", "string"]
-    },
-    "available": {
-      "type": ["null", "boolean"]
-    },
-    "sid": {
-      "type": ["null", "string"]
-    },
-    "actvity_name": {
-      "type": ["null", "string"]
-    },
-    "activity_sid": {
-      "type": ["null", "string"]
-    },
-    "url": {
-      "type": ["null", "string"]
-    },
-    "links": {
-      "type": ["null", "object"]
-    }
-  }
-}
+    "type": "object",
+    "properties": {
+      "account_sid": {
+        "type": ["null", "string"]
+            },
+            "date_created": {
+              "type": ["null", "string"],
+              "format": "date-time"
+            },
+            "date_updated": {
+              "type": ["null", "string"],
+              "format": "date-time"
+            },
+            "workspace_sid": {
+              "type": ["null", "string"]
+            },
+            "attributes": {
+              "type": ["null", "string"]
+            },
+            "date_status_changed": {
+              "type": ["null", "string"],
+              "format": "date-time"
+            },
+            "friendly_name": {
+              "type": ["null", "string"]
+            },
+            "available":{
+              "type": ["null", "boolean"]
+            },
+            "sid": {
+              "type": ["null", "string"]
+            },
+            "actvity_name": {
+              "type": ["null", "string"]
+            },
+            "activity_sid": {
+              "type": ["null", "string"]
+            },
+            "url": {
+              "type": ["null", "string"]
+            },
+            "links": {
+              "type": ["null", "object"]
+            }
+        }
+}
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.